src/adapters/PeopleAdapter.js

```diff
@@ -0,0 +1,13 @@
+import {throwError} from 'rxjs';
+
+/**
+ * This is the base class that `PeopleAdapter` will inherit from.
+ * When creating your subclass `peopleAdapter`, make sure all methods
+ * are defined and the items returned have the proper shape.
+ */
+export default class PeopleAdapter {
+  // eslint-disable-next-line no-unused-vars
+  getPerson(id) {
+    return throwError(new Error('getPerson(id) must be defined.'));
```

Comment:
`getPerson(id) must be defined in PeopleAdapter`

No period (.) at the end, and we should also add the context of where the error is coming from